pennylane_lightning_gpu/src/simulator/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 project(lightning_gpu_simulator)
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CUDA_STANDARD 17)
```

Comment:
Curious when we get full CPP 20 support for nvcc?

Reply:
except for C++ modules: now. https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#cuda-compilers :D

Reply:
Fantastic! Thanks for the update